graphql_compiler/tests/conftest.py

```diff
@@ -50,6 +52,35 @@ def _init_graph_client(load_schema_func, generate_data_func):
     return graph_client
 
 
+@pytest.fixture(scope='session')
+def init_integration_neo4j_client():
+    """Return a client for an initialized db, with all test data imported."""
+    return _init_neo4j_client(generate_neo4j_integration_data)
+
+
+def _init_neo4j_client(generate_data_func):
```

Comment:
What about making a retry decorator with most of the logic here?

Reply:
I like that idea-- will do something similar for graph_client as well (aka orientdb client)

Reply:
Also, I think it makes more sense to keep these two separate since there's no load_schema function for Neo4j but there is for Orientdb.